stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -1180,7 +1180,8 @@ extension String {
   /// values found in the `String`.
   public
   func propertyListFromStringsFileFormat() -> [String : String] {
-    return _ns.propertyListFromStringsFileFormat() as! [String : String]
+    return _ns.propertyListFromStringsFileFormat()! as [NSObject : AnyObject]
+      as! [String : String]
```

Comment:
It's unfortunate. We allow bridging conversions from ObjC <-> Swift, but not a Swift -> ObjC -> Swift round trip cast, which would be necessary to go from `AnyHashable -> NSObject -> String`.